app/budget/directives/budgetCosts/budgetCostsPrint/budgetCostsPrint.css

```diff
@@ -0,0 +1,223 @@
+
+.do-print{
+	display: none;
+}
+
+@media print {
+  @page {
+    margin: 1cm;
+	}
+
+	#toast-container {
+		display: none;
+	}
+	
+	body {
+		font: 9pt "Helvetica Neue", Helvetica, "Noto Sans", sans-serif;
+		line-height: 1.3;
+		}
+  
+	.do-print {
+		display: block !important;
+	}
+
+	p {
+		margin-top: 1px;
+		margin-bottom: 1px; 
+	}
+
+	.page-break-before {
+		page-break-before: always;
+	}
+
+
+	/* SUMMARY REPORT STYLE */
+
+	.budget-summary-print__header-row{
+		border-bottom: 1px solid #ddd;
+	}
+	.budget-summary-print__header-cell {
+		padding-top: 10px;
+		padding-bottom: 10px;
+		text-align: right;
+		font-weight: bold;
+	}
+	
+	.budget-summary-print__header-cell--right {
+		padding-right: 15px;
+	}
+	
+	.budget-summary-print__cell-container {
+		height: 18px;
+		display: flex;
+		align-items: center;
+		justify-content: flex-end;
+	}
+	
+	.budget-summary-print__cell-container--left {
+		padding-left: 5px;
+		justify-content: flex-start;
+	}
+	
+	.budget-summary-print__cell-container--last {
+		padding-right: 5px;
+	}
+	
+	.budget-summary-print-print__table {
+		width: 670px;
+		font: 8.5pt "Helvetica Neue", Helvetica, "Noto Sans", sans-serif;
+		margin-bottom: 20px;
+		table-layout: fixed;
+	}
+	
+	.budget-summary-print__cell--bold {
+		font-weight: bold;
+		margin-bottom: 30px;
+	}
+	
+	.budget-summary-print__cell--total {
+		border-top: 1px solid black;
+	}
+	
+	.budget-summary-print__cell--total-strong {
+		border-top: 3px double black;
+	}
+	
+	.budget-summary-print__cell--funds {
+		font-weight: bold;
+	}
+	
+	.budget-summary-print--total {
+		border-top: 1px solid black;
+		padding-right: 0px;
+		height: 100%;
+		display: flex;
+		align-items: center;
+		justify-content: flex-end;
+	}
+	
+	.budget-summary-print__total-sch {
+		border-top: 1px solid black;
+	}
+	
+	.budget-summary-print__hidden-budgeted-warning {
+		padding: 5px;
+		color: #b1b1b1;
+		text-align: center;
+	}
+	
+
+/* SCHEDULE COST STYLE */
+
+	.budget-costs-print, .budget-costs-print__table{
+		width: 670px !important;
+		font: 8pt "Helvetica Neue", Helvetica, "Noto Sans", sans-serif;
+	}
+
+	.budget-costs-print__table {
+		border-collapse: collapse;
+		font: 7.5pt "Helvetica Neue", Helvetica, "Noto Sans", sans-serif;
+	}
+	
+	.budget-costs-print__table-page-brake {
+		page-break-before: auto;
+		page-break-inside: avoid !important;
+	} 
+
+	.budget-costs-print__table > thead > tr > th,
+	.budget-costs-print__table > tbody > tr > td {
+		vertical-align: center;
+		border: 1px solid #e3e3e3;
+		text-align: center;
+	}
+
+	.budget-costs-print__table > tbody > tr > td {
+		border-top:0;
+	}
+
+	.budget-costs-print__table > tbody > tr > td {
+		padding: 0px !important;
```

Comment:
Only use the important flag as a last resort, can this be re-written without the flag?